app/controllers/metrics_controller.rb

```diff
@@ -51,8 +41,6 @@ def update
     end
   end
 
-  # DELETE /metrics/1
-  # DELETE /metrics/1.json
   def destroy
     @metric.destroy
     respond_to do |format|
```

Comment:
@dcox I'd be concerned about accidental deletions but perhaps protecting from that can wait as a feature to implement later. 

Here's some options to consider for the future:
- Regular database backups
- No delete functionality
- Archive/hide/soft-delete instead of actual deletion
- Version metrics and/or features. Each update is a new version. The old version is archived.
- A user can fork other user's metrics/features and edit their fork rather than edit the original.